### PR TITLE
Add support for iOS 8

### PR DESCRIPTION
**Pull request by insytzsolutions**

### CODE REVIEW

File: SwipeCellKit.podspec [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
    s.summary = 'Swipeable UITableViewCell based on the stock Mail.app, implemented in Swift.'
-   s.homepage = 'https://github.com/SwipeCellKit/SwipeCellKit'
+   s.homepage = 'https://github.com/insytzsolutions/SwipeCellKit'
    s.documentation_url = 'https://swipecellkit.github.io/SwipeCellKit/'
    s.social_media_url = 'https://twitter.com/mkurabi'
    s.author = 'Mohammad Kurabi'
 
-   s.source = { :git => 'https://github.com/SwipeCellKit/SwipeCellKit.git', :tag => s.version }
+   s.source = { :git => 'https://github.com/insytzsolutions/SwipeCellKit.git', :tag => s.version }
```
Comment (ignatovSA) [inline SwipeCellKit.podspec]: Please revert the change of pod source repo.

**Comment by kurabi** [comment]
Thank you for the PR. This has already been submitted previously in another PR and the decision was not to support iOS 8. iOS 12 is around the corner so support for 4 major versions worth is plenty enough.
